packages/block/src/block.ts

```diff
@@ -52,6 +53,14 @@ export class Block {
     return trie.root()
   }
 
+  /**
+   * Returns the withdrawals ssz root for array of Withdrawal.
+   * @param wts array of Withdrawal to compute the root of
+   */
+  public static async genWithdrawalsSSZRoot(wts: Withdrawal[]) {
```

Comment:
```suggestion
  public static async generateWithdrawalsSSZRoot(withdrawals: Withdrawal[]) {
```

The naming convention for the short version isn't 100% clear to me.  Prefer this.